### PR TITLE
French translation for CSS `aspect-ratio` property reference.

### PR DESCRIPTION
**Pull request by audrasjb**
This PR aims to create a French translation for CSS `aspect-ratio` property reference.

### CODE REVIEW

File: files/fr/web/css/aspect-ratio/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+<h3 id="examples_of_values_for_aspect-ratio">Exemples de valeurs pour aspect-ratio</h3>
+
+<pre class="brush: css">aspect-ratio: 1 / 1;
+aspect-ratio: 16 / 9;
+</pre>
+
+<h2 id="mapping_width_and_height_to_aspect-ratio">Correspondance entre largeur et hauteur avec aspect-ratio</h2>
+
+<p>Les navigateurs ont ajouté une propriété <code>aspect-ratio</code> interne qui s'applique aux <a href="/fr/docs/Web/CSS/Replaced_element">éléments remplacés</a> et à d'autres éléments liés qui acceptent les attributs <code>width</code> et <code>height</code>. Cela se passe dans la feuille de styles interne UA du navigateur.</p>
```
Comment (SphinxKnight) [inline files/fr/web/css/aspect-ratio/index.html:47]: It seems to me that "feuille de styles interne UA du navigateur" is a bit superfluous/complex vs "feuille de styles interne du navigateur" (as the User Agent is the browser anyway).
If there are various internal stylesheets, please discard my comments.

**Review by SphinxKnight — CHANGES_REQUESTED** [review]

File: files/fr/web/css/aspect-ratio/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/aspect-ratio/index.html:47]: LGTM

File: files/fr/web/css/aspect-ratio/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+</pre>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+ <dt>{{cssxref("&lt;auto&gt;")}}</dt>
+ <dd><a href="/en-US/docs/Web/CSS/Replaced_element">Replaced elements</a> with an intrinsic aspect ratio use <em>that</em> aspect ratio, otherwise the box has no preferred aspect ratio. Size calculations involving intrinsic aspect ratio always work with the content box dimensions.</dd>
+ <dt>{{cssxref("&lt;ratio&gt;")}}</dt>
+ <dd>The box’s preferred aspect ratio is the specified ratio of <code>width</code> / <code>height</code>. Size calculations involving preferred aspect ratio work with the dimensions of the box specified by <code>box-sizing</code>.</dd>
+</dl>
```
[the remark this answers is not in the document]
Comment (audrasjb) [reply files/fr/web/css/aspect-ratio/index.html:27]: oops 🤭

[the remark this answers is not in the document]
Comment (audrasjb) [reply files/fr/web/css/aspect-ratio/index.html]: Done in https://github.com/mdn/translated-content/pull/1722/commits/91db71aaecc43376852a5c0c93fb01acaabcd47c

File: files/fr/web/css/aspect-ratio/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/aspect-ratio/index.html]: Fixed in https://github.com/mdn/translated-content/pull/1722/commits/91db71aaecc43376852a5c0c93fb01acaabcd47c

**Review by SphinxKnight — APPROVED** [review]

**Pull request merged**